learnClass/learnClass.js

```diff
@@ -0,0 +1,81 @@
+class Student {
+    constructor(name, age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    getAge() {
+        return this.age
+    }
+
+    getGrade() {
+        if (this.age > 5) {
+            return "school going kid"
+        } else {
+            return "kindergarden kid"
+        }
+
+    }
+}
+
+class Grade5Students extends Student {
+    constructor(name, age, percentage) {
+        super(name, age)
+        this.percentage = percentage;
+    }
+
+    getPercentage() {
+        return this.percentage;
+    }
+
+    getGrade() {
+        console.log(super.getGrade());
+        return 5;
+    }
+}
+
+let resh = new Student("resh", 6)
+let skan = new Student("Skandhana", 1)
+let jahn = new Grade5Students("jahn", 12, 80)
+console.log(jahn.getGrade())
+console.log(resh.getGrade())
+console.log(skan.getGrade())
```

Comment:
Remove this code from here. Write unit tests for this.

Reply:
done